samples/meetings-sidepanel/csharp/Side Panel/Model/sidePanel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SidePanel.Models
+{
+    public class sidePanel
+    {
```

Comment:
Please check and remove if this class is not being used anywhere.

Reply:
I see that this is not used, could you please help to remove this?
![image](https://user-images.githubusercontent.com/31851992/115446020-3bc99d80-a234-11eb-9717-913f6c752218.png)